ipalib/cli.py

```diff
@@ -680,10 +682,39 @@ def select_entry(self, entries, format, attrs, display_count=True):
         self.print_line('')
         return selection
 
+
 class help(frontend.Local):
     """
     Display help for a command or topic.
     """
+    class Writer:
```

Comment:
Maybe adding 'object' inheritance could stop pylint from whining.

Reply:
It was not the case but thanks anyway :)